### DISCUSSION TITLE
Decouple particle emission cycle from lifetime

### DISCUSSION THREAD

**Discussion by timothyqiu** [godotengine/godot-proposals#5569] [General Discussions]
It seems impossible to make a Particles node to emit 2 particles in every 10 seconds and each particle exists for only 1 second.

**Comment by Calinou** [comment]
> to emit 2 particles in every 10 seconds 

For this kind of use case where high timing precision isn't required, using a Timer node along with the particles may be a better idea.

**Reply by timothyqiu** [reply]
Yeah, I also saw this approach being suggested in the Q&A site.

I've been feeling something strange when setting up particle nodes for a long time, and was a bit surprised when I finally realized that the particle lifetime also controls the emission cycle.

Most particle systems I've seen separate these two properties. It helps to build a clear mental model of the particle system I think.
